core/src/KokkosExp_MDRangePolicy.hpp

```diff
@@ -182,6 +182,7 @@ struct MDRangePolicy : public Kokkos::Impl::PolicyTraits<Properties...> {
   point_type m_tile_end       = {};
   index_type m_num_tiles      = 1;
   index_type m_prod_tile_dims = 1;
+  bool m_tune_tile_size       = false;
```

Comment:
You need to update
https://github.com/kokkos/kokkos/blob/becce485d0cb51cb56f054d51344b4286d567a3d/core/src/KokkosExp_MDRangePolicy.hpp#L269-L277

Reply:
Good catch

Reply:
Addressed via 03a0446